sdk/eventhub/eventhubs-checkpointstore-blob/package.json

```diff
@@ -40,12 +40,13 @@
     "execute:samples": "dev-tool samples run samples-dev",
     "extract-api": "tshy && dev-tool run extract-api",
     "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.{ts,cts,mts}\" \"test/**/*.{ts,cts,mts}\" \"samples-dev/**/*.{ts,cts,mts}\" \"*.{js,cjs,mjs,json}\"",
-    "integration-test:browser": "tshy && dev-tool run build-test && cross-env TEST_MODE=live dev-tool run test:vitest --browser --no-test-proxy",
+    "integration-test:browser": "tshy && dev-tool run build-test && npm run playwright-install && cross-env TEST_MODE=live dev-tool run test:vitest --browser --no-test-proxy",
```

Comment:
can we make it part of the `dev-tool run test:vitest --browser` command instead?

Reply:
There's https://github.com/Azure/azure-sdk-for-js/blob/fe53912d570592b8cd2bd3dd872b5b76d7f7bf0e/common/tools/dev-tool/src/commands/run/testVitest.ts#L52 

For some reason it didnt work?